config_defaults_inc.php

```diff
@@ -2085,6 +2085,28 @@
 $g_bug_closed_status_threshold = CLOSED;
 
 /**
+ * Bug becomes closed for submission of automated bug reports (attaches)
+ * if its status is >= $g_bug_collect_reports_threshold. The bug starts 
+ * accept more bug reports again if its status becomes less than this 
+ * threshold. The default is ASSIGNED. 
```

Comment:
Default is not consistent with value below.  Typically we don't mention the default in config_defaults_inc since it right under it.  But we do in the configuration section of the manual.